target/linux/generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch

```diff
@@ -10,26 +10,6 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 
 --- a/drivers/net/ethernet/mediatek/mtk_eth_soc.c
 +++ b/drivers/net/ethernet/mediatek/mtk_eth_soc.c
-@@ -3155,8 +3155,8 @@ static irqreturn_t mtk_handle_irq_rx(int
- 
- 	eth->rx_events++;
- 	if (likely(napi_schedule_prep(&eth->rx_napi))) {
--		__napi_schedule(&eth->rx_napi);
- 		mtk_rx_irq_disable(eth, eth->soc->txrx.rx_irq_done_mask);
-+		__napi_schedule(&eth->rx_napi);
- 	}
- 
- 	return IRQ_HANDLED;
-@@ -3168,8 +3168,8 @@ static irqreturn_t mtk_handle_irq_tx(int
- 
- 	eth->tx_events++;
- 	if (likely(napi_schedule_prep(&eth->tx_napi))) {
--		__napi_schedule(&eth->tx_napi);
- 		mtk_tx_irq_disable(eth, MTK_TX_DONE_INT);
-+		__napi_schedule(&eth->tx_napi);
- 	}
- 
- 	return IRQ_HANDLED;
```

Comment:
LOL... no idea we had this change... I accidentally pushed the same change while reworking napi_schedule and noticing the problem in how it was handled.

Reply:
How shall we proceed?  Is this PR good as-is or do you suggest some change?

Reply:
Ok as is. It just got backported. Was just me surprised noticing this and wanted to put some text about it ahhahah

Reply:
Sounds good, thanks!